README.txt

```diff
@@ -0,0 +1,19 @@
+# Guinness
+
+## Set Up
+
+### `.env`
+
+The Engineer
+
+## Dependencies
+
+A liver
+
+## How to contribute
+
+1. Read our Guiness Challenge Style Guide on Notion
+2. Go to The Engineer after Harker Tech every week
+3. Do the Guiness Challenge in line with our style guide
+4. Upload a picture of your challenge result with the `.JPEG` file extension (`.JPEG` specifically, none of this cop-out `.JPG` behaviour)
```

Comment:
@faizanabbasjsc we decided at the All Hands on 25/06/2024 that the standard for Guinness attempts will be GIFs. Please amend accordingly. It's worth keeping in the details about JPEGs anyway, just to idiot-proof it.

Reply:
Apologies this slipped my mind, I shall make this change with haste.